### ISSUE TITLE
Failed to extract `d_c0` from cookies for zhihu

### ISSUE THREAD

**Issue by YukiSou** [DIYgod/RSSHub#16629]
### Routes

```routes
/zhihu/people/activities/:id
```

### Full routes

```fullroutes
/zhihu/people/activities/the-41-92
```

### Related documentation

https://docs.rsshub.app/routes/social-media#%E7%94%A8%E6%88%B7%E5%8A%A8%E6%80%81-1

### What is expected?

working

### What is actually happening?

not working
<img width="1015" alt="Screenshot 2024-09-05 at 10 45 17 AM" src="https://github.com/user-attachments/assets/c62b02a7-25b7-4ece-a336-080748695ad3">

### Deployment information

Self-hosted

### Additional info

```shell
Node Version: v22.5.1

Git Hash: 2f3cff8d

Git Date: Wed, 04 Sep 2024 23:22:43 GMT
```

### This is not a duplicated issue

- [X] I have searched [existing issues](https://github.com/DIYgod/RSSHub/issues) to ensure this bug has not already been reported

**Comment by dzx-dzx** [comment]
`d_c0` is not read from the configuration, but obtained by sending requests. This is a volatile process, especially, as I've observed, when the host is located outside of China.

**Comment by AntoineChn** [comment]
我也遇到了这个问题，有什么进展么？

> d_c0 is not read from the configuration, but obtained by sending requests. This is a volatile process, especially, as I've observed, when the host is located outside of China.

我将服务器迁移至国内，仍然不能避免。

**Comment by Anillc** [comment]
同样有这个问题，看上去 #16691 在修复这个问题

**Comment by TonyRL** [comment]
Closed in #16691
